test/framework/tutorials/fv_test1.cc

```diff
@@ -112,12 +112,14 @@ SimTest01_FV(const InputParameters&)
   // Create Krylov Solver
   opensn::log.Log() << "Solving: ";
   auto petsc_solver =
-    CreateCommonKrylovSolverSetup(A, "FVDiffSolver", KSPCG, PCGAMG, 0.0, 1.0e-9, 1000);
+    CreateCommonKrylovSolverSetup(A, "FVDiffSolver", KSPCG, PCGAMG, 0.0, 1.0e-15, 25);
```

Comment:
For my own edification, why bump the tolerance to 1e-15 here?

Reply:
That's most likely a left-over from when I was looking at the PETSc 3.17 vs 3.19 and tried to converged some failing tests more to see if they would pass.